internal/pkg/manifest/pipeline.go

```diff
@@ -17,6 +17,7 @@ const (
 	GithubV1ProviderName   = "GitHubV1"
 	CodeCommitProviderName = "CodeCommit"
 	BitbucketProviderName  = "Bitbucket"
+	DefaultImage           = "aws/codebuild/amazonlinux2-x86_64-standard:3.0"
```

Comment:
Can we define this constant in the`deploy` pkg instead? this would allow us to keep it private